.azuredevops/publish-platform-nginxplus.yaml

```diff
@@ -48,6 +49,13 @@ extends:
         developmentEnvironment: true
         azureRegions:
           primary: 'UKSouth'
+      - name: 'ssv5'
+        serviceConnection: $(ssv5Subscription)
+        deploymentBranches:
+          - 'refs/heads/main'
+        developmentEnvironment: true
+        azureRegions:
+          primary: 'UKSouth'
     scriptsList:
```

Comment:
-AzureSubscription $(ssv3Subscription). ssv3Subscription is hardcoded? This will not work for ssv5 env, right?

Reply:
I have fixed it and updated -AzureSubscription param to use $(subscriptionName) from var file